### PR TITLE
add limitations.md to docsite

### PR DESCRIPTION
**Pull request by GavT**
### Why is this change needed?

- developers are hitting limitations of Testnet without knowing it is a limitation and so potentially wasting time working around it

### What changes were made as part of this PR:

- new limitations.md file
- pulled in Cais's changes to Introduction.md

### What are the key areas to look at

- limitations.md
- introduction.md

### :rotating_light: Definition of Done :rotating_light:
- [ ] Merged into dev-testnet 
- [ ] Passes tests on dev-testnet
- [ ] [Changelog.md](https://github.com/obscuronet/go-obscuro/blob/main/docs/testnet/changelog.md) updated

### CODE REVIEW

File: docs/_data/navigation.yml [hunk 1]
```diff
@@ -27,6 +27,8 @@ pages:
         url: testnet/example-dapps.html
       - title: Essential Information
         url: testnet/essentials.html
+      - title: Limitations
```
Comment (tudor-malene) [inline docs/_data/navigation.yml:30]: maybe "Known limitations"?

File: docs/testnet/Introduction.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-All Obscuro code is intentionally under an open source licence. If you see something you think could be improved or reworked feel free to submit a PR in the [go-obscuro repo](https://github.com/obscuronet/go-obscuro), or to let us know about your suggestion on [Discord](https://discord.com/channels/916052669955727371/945360340613484684). 
+All Obscuro code is under an open source licence. If you see something you think could be improved or reworked feel free to submit a PR in the [go-obscuro repo](https://github.com/obscuronet/go-obscuro), or to let us know about your suggestion on [Discord](https://discord.com/channels/916052669955727371/945360340613484684). 
 
-As you would expect from a Testnet, OBX tokens issued on Obscuro Testnet have no value. There are no gas fees, for now at least. As Testnet develops, gas fees will be introduced along with an OBX faucet.
+OBX tokens issued on Obscuro Testnet have no real value and gas fees are initially set to zero. As Testnet develops, gas fees will be introduced along with an OBX faucet.
```
Comment (tudor-malene) [inline docs/testnet/Introduction.md:12]: "Native" OBX...

File: docs/testnet/limitations.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Testnet Limitations
+The vision for Obscuro is to be fully compatible with the Ethereum Virtual Machine and provide an entirely decentralised privacy experience. Testnet goes a long way to achieving that but there is more to be done and for now there are some limitations. Over time the number of limitations will decrease.
+
+These are the key limitations to be aware of when developing for Obscuro testnet:
+
+1. Streaming events are not yet implemented. Instead events are only available in the transaction receipt.
+1. The gas feature is not implemented yet.
+1. Native value transfer is not implemented yet.
```
Comment (tudor-malene) [inline docs/testnet/limitations.md:8]: it is now.

File: docs/testnet/limitations.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+1. New nodes cannot be added. For now the number of Obscuro nodes is fixed.
+1. Data revelation is not implemented yet.
```
Comment (tudor-malene) [inline docs/testnet/limitations.md:10]: 1. Security is not fully implemented. Some keys are still hardcoded.
2. The decentralised bridge is limited to two hardcoded ERC20 tokens.
3. The Layer 1 is currently a hosted network. In the next iteration we'll connect Obscuro to Ropsten.
4. The "Wallet Extension" is not fully polished yet.

**Review by tudor-malene — COMMENTED** [review]
looks good
added a few more limitations.

File: docs/_data/navigation.yml (hunk 1 above)
Comment (GavT) [reply docs/_data/navigation.yml:30]: Yep

File: docs/testnet/Introduction.md (hunk 2 above)
Comment (GavT) [reply docs/testnet/Introduction.md:12]: ok

File: docs/testnet/limitations.md (hunk 4 above)
Comment (GavT) [reply docs/testnet/limitations.md:10]: OK. But Goerli, not Ropsten

File: docs/testnet/limitations.md (hunk 3 above)
Comment (GavT) [reply docs/testnet/limitations.md:8]: cool

**Review by GavT — COMMENTED** [review]
Updated following comments from Tudor

**Review by tudor-malene — APPROVED** [review]
lgtm

**Pull request merged**